components/button-group/css/template.tsx

```diff
@@ -2,23 +2,24 @@
  * @license EUPL-1.2
  * Copyright (c) 2022 Community for NL Design System
  */
+import { ButtonGroup as ButtonGroupNL } from '@utrecht/component-library-react';
```

Comment:
I suggest to rename `ButtonGroupNL` to `UtrechtButtonGroup`, because we want to reserve the prefix NL for the "hall of fame" candidates.